docs/ConfigureScalarDL.md

```diff
@@ -215,6 +219,100 @@ auditor:
 
 To enable the Auditor service, you can follow the [Enable Auditor](#enable-auditor) section.
 
+### JDBC Databases
```

Comment:
I think we need to specify the supported JDBC database here

Reply:
Updated with the list of JDBC databases.